tcf_website/templates/course/course_professor.html

```diff
@@ -164,24 +184,33 @@ <h3 class="font-weight-light text-tcf-orange">
         </div>
 
         <div class="reviews-container mb-5 px-3">
-            <div class="reviews-toolbar d-flex">
-                <div class="ml-auto dropdown btn-group">
-                    <button class="btn bg-white" disabled>Sort by</button>
-                    <button id="review-sort-select" class="btn btn-primary dropdown-toggle" type="button" id="sortOptions"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></button>
-                    <div class="dropdown-menu" aria-labelledby="sortOptions">
-                        <button id="votes-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Most Helpful
-                        </button>
-                        <button id="recent-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Most Recent
-                        </button>
-                        <button id="highrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Highest Rating
-                        </button>
-                        <button id="lowrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Lowest Rating
-                        </button>
+            <div class="reviews-toolbar d-flex flex-md-row flex-column justify-content-left justify-content-md-center">
+                <div class="reviews-header d-inline-flex">
+                    <h3 class="h2">{{ reviews|length }} Comments</h3>
+
+                </div>
+                <div class="ml-auto">
+                    <a href="{% url 'new_review' %}" class="btn bg-tcf-orange text-white mr-4">
+                      <i class="fa fa-plus" aria-hidden="true"></i> Add your review!
+                    </a>
+                    <div class="dropdown btn-group">
+                        <button class="btn bg-white" disabled>Sort by</button>
+                        <button id="review-sort-select" class="btn btn-primary dropdown-toggle" type="button" id="sortOptions"
+                                data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></button>
+                        <div class="dropdown-menu" aria-labelledby="sortOptions">
```

Comment:
There's no default sort if there's 0 reviews
![image](https://user-images.githubusercontent.com/45245651/102024987-19de7c80-3d63-11eb-8bac-060314b1d047.png)

Reply:
oh shit hmm yeah I'll just add a quick if statement for that